src/extend.rs

```diff
@@ -1,11 +1,25 @@
+//! Structured extensions for building and parsing Thing Descriptions
+//!
+//! The Thing Description can be extended adding additional ontologies in its [@context](https://www.w3.org/TR/json-ld11/#the-context).
+//!
+//! This module provides a trait, [ExtendableThing], to define extensions for each of the standard
+//! elements of a description.
+
 use serde::{Deserialize, Serialize};
 
 use crate::hlist::{Cons, Nil};
 
+/// Requirement trait for extending a Thing Description alement
+///
+/// The structure must be serializable/deserializable use `#[serde(skip)]` if it has to hold data
+/// you do not want to push in the serialized description.
```

Comment:
This is unclear to me, it also seem not relevant for trait.
I would just say that `ExtendablePiece` is a sort of _trait alias_ to avoid repeating the serde traits all around.

Reply:
Sounds good. Description removed.